packages/mobile/test/schemas.ts

```diff
@@ -560,6 +560,7 @@ export const v16Schema = {
     },
     exchangeRate: undefined,
     fetchRateFailed: false,
+    error: false,
```

Comment:
We shouldn't touch previous test schemas. They are useful to test migrations and keep track of how it changed over time.
We can either make the new field optional and avoid this. Or create a new schema.

Reply:
There are new schemas in main, so if we cherry pick it will probably cause issues, which is why I added it this way. Figured a simple `error` boolean field wouldn't cause issues but I can make it optional, would have just preferred to avoid it since it isn't necessary, but not a big deal I guess